client/components/playground/timeUtils.ts

```diff
@@ -0,0 +1,31 @@
+export const FIVE_SECONDS = 5000;
+export const ONE_MINUTE = 60000;
+export const ONE_HOUR = ONE_MINUTE * 60;
+export const ONE_DAY = ONE_HOUR * 24;
+export const ONE_MONTH = ONE_DAY * 30;
+export const THREE_MONTHS = ONE_MONTH * 3;
+
+export const timeAgoLabel = (dateString: string): string => {
```

Comment:
This function looks quite complex, could you add some unit tests for it?

Reply:
Done, thanks :)